test/async-hooks/test-statwatcher.js

```diff
@@ -41,19 +49,27 @@ checkInvocations(statwatcher1, { init: 1 },
 checkInvocations(statwatcher2, { init: 1 },
                  'watcher2: when started to watch second file');
 
-// remove first file watcher
-fs.unwatchFile(__filename);
-checkInvocations(statwatcher1, { init: 1, before: 1, after: 1 },
-                 'watcher1: when unwatched first file');
-checkInvocations(statwatcher2, { init: 1 },
-                 'watcher2: when unwatched first file');
+setTimeout(() => fs.writeFileSync(file1, 'foo++'), 10);
```

Comment:
Could this be `setImmediate` or is that unreliable?

Reply:
If we can't use that, shouldn't we be using the platform timeout modifier?

Reply:
So… the reason for this is that the stat watcher itself internally itself uses file system requests + timeouts. In  order to witness a change, the write needs to hit a spot after the initial internal `stat` request has completed, but that’s timing that we can’t really track.

I’ve updated to use `common.platformTimeout(100)`.